### PR TITLE
Create gertrudefile.md

### PR DESCRIPTION
**Pull request by gertrudeln**

### CODE REVIEW

File: participants/gertrudefile.md [hunk 1]
```diff
@@ -0,0 +1 @@
+added this
```
Comment (inful) [inline participants/gertrudefile.md:1]: Yes, but is it your name?

Comment (gertrudeln) [reply participants/gertrudefile.md:1]: was just FAFOing around as usual

Comment (danielvo) [inline participants/gertrudefile.md:1]: Well added!

**Review by danielvo — APPROVED** [review]
Wonderful!

**Pull request merged**